.github/workflows/docs-keywords-filter.yml

```diff
@@ -0,0 +1,76 @@
+name: docs-keywords-filter
```

Comment:
@mmaoyu Thanks for PR this looks good, one suggestion is could make this action as part of `doc-check.yml` CI.

IMO, this should be called as a part of `doc-check.yaml` Ci test?